ssl/ssl_ciph.c

```diff
@@ -1311,7 +1311,8 @@ static int ciphersuite_cb(const char *elem, int len, void *arg)
     cipher = ssl3_get_cipher_by_std_name(name);
     if (cipher == NULL) {
         SSLerr(SSL_F_CIPHERSUITE_CB, SSL_R_NO_CIPHER_MATCH);
-        return 0;
+        /* Ciphersuite not found but return 1 to parse rest of the list */
+        return 1;
```

Comment:
Why is it that this instance returns 0, but the other "no cipher match" failure in this function a few lines above returns 0. Shouldn't that one also return 1?

Reply:
Yes, it should return 1.